backend/api/Startup.cs

```diff
@@ -138,6 +141,20 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IKeycloakRequestClient, KeycloakRequestClient>();
 
             services.AddAutoMapper(typeof(Startup));
+
+            services.AddHealthChecks()
+                .AddCheck("liveliness", () => HealthCheckResult.Healthy())
+                .AddSqlServer(builder.ConnectionString, tags: new[] { "services" });
+            //TODO: Add a health check for keycloak connectivity.
+            services.AddHealthChecksUI(setupSettings: setup =>
+            {
+                setup.AddHealthCheckEndpoint("liveliness", "http://localhost/live");
```

Comment:
While these are hardcoded, I think that is OK since our pims api server would always also be hosting the health check ui interface.

Reply:
1) Make sure all routes are `/api/...`
2) I don't understand the `http://localhost/live` part.  I would think this needs to be configurable, handle https, host and port.

Reply:
/live is the endpoint that I've configured below.
/live is hosted on our server. however, healthchecksui doesn't know that. HealthChecksUI can be configured to run locally on the same server as the health checks, or remotely. in this case we are running locally, thus the http://localhost. According to the docs just having just having /live works in some cases, but they still have some issues (it didn't work for me within docker).

Https should work, but also shouldn't matter since the connection isn't crossing a network.
Same with being configurable, that shouldn't be necessary unless we plan on having HealthChecksUI not run on the same server. If that is the case then we could create a separate container, and then there are completely different connection parameters that can be used to detect health endpoints on kubernetes.

Reply:
I've looked into this a bit more, while I don't want to get hung up on the details too much for this SPIKE, we could configure most of the health checks ui within appsettings.json, for example:

{
  "HealthChecksUI": {
    "HealthChecks": [
      {
        "Name": "HTTP-Api-Basic",
        "Uri": "http://localhost:6457/healthz"
      }
    ],
    "Webhooks": [
      {
        "Name": "",
        "Uri": "",
        "Payload": "",
        "RestoredPayload": ""
      }
    ],
    "EvaluationTimeInSeconds": 10,
    "MinimumSecondsBetweenFailureNotifications": 60
  }
}
we would still need to inject the value of the webhook secret here, but this might be more what you are looking for.